matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/DefaultFileService.kt

```diff
@@ -96,6 +97,9 @@ internal class DefaultFileService @Inject constructor(
             }
         }
 
+        var atomicFileCreator1: AtomicFileCreator? = null
```

Comment:
Can it be renamed to something like encyptedCachedAtomicFile

Reply:
Renamed to `atomicFileDownload`, it can be a clear file